src/lib/FlightTasks/tasks/AutoMapper/FlightTaskAutoMapper.cpp

```diff
@@ -86,6 +86,8 @@ bool FlightTaskAutoMapper::update()
 		_generateVelocitySetpoints();
 	}
 
+	_obstacle_avoidance.updateAvoidanceDesiredSetpoints(_position_setpoint, _velocity_setpoint);
```

Comment:
Is it necessary/desired to publish these all the time even if there is no obstacle avoidance enabled or present?
Could it be conditioned by `COM_OBS_AVOID` either here or inside the `updateAvoidanceDesiredSetpoints()` function?